src/corelibs/U2Algorithm/src/misc/EnzymeModel.h

```diff
@@ -52,12 +52,14 @@ class DNAAlphabet;
 
 #define ENZYME_CUT_UNKNOWN 0x7FFFFF
 #define ENZYME_LIST_SEPARATOR ","
+#define SUPPLIERS_LIST_SEPARATOR ";"
```

Comment:
q: is `;` can be a part of the supplier name? If yes, change the separator to `\n`